src/coreComponents/mesh/generators/MeshGeneratorBase.hpp

```diff
@@ -100,9 +79,15 @@ class MeshGeneratorBase : public dataRepository::Group
    * @param[in] cellBlockManager to fill with the mesh informations
    * @return The global length scale
    */
-  virtual MeshGeneratorHelper generateCellBlockManager( CellBlockManager & cellBlockManager ) = 0;;
+  virtual void generateCellBlockManager( CellBlockManager & cellBlockManager ) = 0;
```

Comment:
I think this member function can be declared `private`.
And be careful of the `doxygen` comments.